Sources/SWBCore/SpecImplementations/Tools/SwiftCompiler.swift

```diff
@@ -2755,7 +2754,7 @@ public final class SwiftCompilerSpec : CompilerSpec, SpecIdentifierType, SwiftDi
             moduleDirPath = scope.evaluate(BuiltinMacros.BUILT_PRODUCTS_DIR).join(moduleName + ".swiftmodule")
         }
         if isProject {
-            // Copy this content to the Project subdir so we could master them out when installing.
```

Comment:
nit: "exclude them by filename pattern" might be more accurate to the original term-of-art meaning